docs/docs/sourcing-from-flotiq.md

```diff
@@ -0,0 +1,264 @@
+
+> You can find more details about how to configure the Flotiq source plugin in the [plugin's README](https://github.com/flotiq/gatsby-source-flotiq).
+
+## Accessing Flotiq data in Gatsby
+
+Once you finish configuring your environment you can start developing. The source plugin will pull all the content from your Flotiq account and create respective Gatsby GraphQL nodes. It will also preserve the relations you've setup in Flotiq, so the GraphQL nodes will be automatically linked.
+
+For example, if you define a relation between `BlogPost` and `Category` content types in Flotiq - they will retain their relationship in Gatsby's GraphQL, so you can retrieve that in a query:
+
+```graphql
+query {
+  allBlogPost {
+    nodes {
+      id
+      slug
+      category {
+        name
+      }
+    }
+  }
+}
+```
+
+The next step is implementing these queries in your `gatsby-node.js` file:
+
+```javascript:title=gatsby-node.js
+const path = require(`path`)
+const { createFilePath } = require(`gatsby-source-filesystem`)
+exports.createPages = async ({ graphql, actions }) => {
+  const { createPage } = actions
+
+  const blogPost = path.resolve(`./src/templates/blog-post.js`)
+  const result = await graphql(`
+    query GetBlogPosts {
+      allBlogpost(sort: { fields: flotiqInternal___createdAt, order: DESC }) {
+        edges {
+          node {
+            headerImage {
+              extension
+              id
+            }
+            content
+            id
+            slug
+            title
+          }
+        }
+      }
+    }
+  `)
+
+  if (result.errors) {
+    throw result.errors
+  }
+
+  // Create blog posts pages.
+  const posts = result.data.allBlogpost.edges
+  posts.forEach((post, index) => {
+    const previous = index === posts.length - 1 ? null : posts[index + 1].node
+    const next = index === 0 ? null : posts[index - 1].node
+
+    createPage({
+      path: post.node.slug,
+      component: blogPost,
+      context: {
+        slug: post.node.slug,
+        previous,
+        next,
+      },
+    })
+  })
+}
+
+exports.onCreateNode = ({ node, actions, getNode }) => {
+  const { createNodeField } = actions
+
+  if (node.internal.type === `MarkdownRemark`) {
+    const value = createFilePath({ node, getNode })
+    createNodeField({
+      name: `slug`,
+      node,
+      value,
+    })
+  }
+}
+```
+
+Now you'll want to let Gatsby create appropriate pages for your Content Objects. This example uses a `blogPost` component as a template, which can also be found below:
+
+```javascript:title=blog-post.js
+import React from "react"
+import { Link, graphql } from "gatsby"
+
+import { rhythm, scale } from "../utils/typography"
```

Comment:
```suggestion
```
This will fail since the directory doesn't exist, so I'd recommend removing it to simplify.